core/src/test/java/org/apache/accumulo/core/client/impl/TableOperationsHelperTest.java

```diff
@@ -183,6 +183,7 @@ public void setLocalityGroups(String tableName, Map<String,Set<Text>> groups)
     }
 
     @Override
+    @SuppressWarnings("deprecation")
```

Comment:
Might be better to mark this method as deprecated itself... rather than suppress the warning. Marking this deprecated makes it clear that this exists to test the deprecated method, and will be removed when the deprecated method is removed.